code/obj/item/staffkiosk.dm

```diff
@@ -0,0 +1,287 @@
+	dispensedKit = /obj/item/storage/box/staffkit/chef
+
+/datum/recruitment_role/bar
+	name = "Bar Assistant"
+	accessParent = "Bartender"
+	canAuthorize = list("Bartender","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-bar"
+	dispensedKit = /obj/item/storage/box/staffkit/bar
+
+/datum/recruitment_role/botany
+	name = "Botanical Assistant"
+	accessParent = "Botanist"
+	canAuthorize = list("Botanist","Head of Personnel","Captain")
+	cardIcon = "id_civ"
+	tokenIcon = "assist-token-hyd"
+	dispensedKit = /obj/item/storage/box/staffkit/botany
+
+
+//equipment kits
+
+/obj/item/storage/box/staffkit/med
+	name = "medical assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/reagent_containers/emergency_injector/epinephrine,
+		/obj/item/reagent_containers/mender/brute,
+		/obj/item/reagent_containers/mender/burn,
+		/obj/item/bandage)
+
+/obj/item/storage/box/staffkit/robo
+	name = "robotics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffmed,
+		/obj/item/device/radio/headset/medical,
+		/obj/item/disk/data/cartridge/medical,
+		/obj/item/screwdriver,
+		/obj/item/cable_coil,
+		/obj/item/body_bag)
+
+/obj/item/storage/box/staffkit/eng
+	name = "engineering assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/engineer,
+		/obj/item/clothing/head/helmet/hardhat,
+		/obj/item/crowbar,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
+
+/obj/item/storage/box/staffkit/mech
+	name = "mechanics assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/disk/data/cartridge/mechanic,
+		/obj/item/electronics/soldering,
+		/obj/item/device/t_scanner,
+		/obj/item/screwdriver,
+		/obj/item/device/multitool)
+
+/obj/item/storage/box/staffkit/miner
+	name = "mining assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffeng,
+		/obj/item/device/radio/headset/engineer,
+		/obj/item/clothing/head/emerg,
+		/obj/item/clothing/suit/space/emerg,
+		/obj/item/crowbar,
+		/obj/item/mining_tool/power_pick,
+		/obj/item/satchel/mining)
+
+/obj/item/storage/box/staffkit/res
+	name = "research assistant kit"
+	desc = "A slightly stale-smelling set of equipment."
+	spawn_contents = list(/obj/item/clothing/under/rank/staffres,
+		/obj/item/device/radio/headset/research,
+		/obj/item/clothing/mask/gas,
+		/obj/item/clothing/gloves/latex,
+		/obj/item/reagent_containers/glass/beaker,
+		/obj/item/device/reagentscanner)
```

Comment:
`/obj/item/device/reagentscanner` should be readily available once they make it to the department

Reply:
Reagent scanners are also standard and required engineering equipment and it would make no sense not to include them, they already have to get an upgrade in the department to actually check the combustion chamber.

Reply:
Think I'm fine with this item complement, on the whole. There do tend to be spares of reagent scanners on most maps, but only one or two, and they're markedly inferior compared to spectro scanner goggles so I don't think it's unreasonable to give them out with the kit.